pages/common/fastmod.md

```diff
@@ -0,0 +1,29 @@
+# fastmod
+
+> A fast partial replacement for the codemod tool, replace and replace all in the whole codebase.
+> Regexes are matched by Rust regex crate.
+> More information: <https://github.com/facebookincubator/fastmod>.
+
+- Replace regex pattern in all files of the current directory, ignoring files on .ignore and .gitignore:
+
+`fastmod {{regex_pattern}} {{replacement}}`
+
+- Case-insensitively replace regex pattern in specific files even if ignored, or directory ignoring files on .ignore and .gitignore:
+
+`fastmod --ignore-case {{regex_pattern}} {{replacement}} -- {{file}} {{dir/}} {{...}}`
```

Comment:
```suggestion
`fastmod --ignore-case {{regex_pattern}} {{replacement}} -- {{path/to/file1}} {{path/to/file2}} {{path/to/directory}}`
```

Reply:
`...` is more concise and meaningful. Can I keep it?

Reply:
:thinking: we use this for enumerations:
```suggestion
`fastmod --ignore-case {{regex_pattern}} {{replacement}} -- {{path/to/file_or_directory1 path/to/file_or_directory2 ...}}`
```